src/test/java/org/apache/drill_web_test_framework/rest_api/tests/unsecure/StorageTest.java

```diff
@@ -216,27 +225,25 @@ public void queryUpdatedTestPlugin() {
         .post("/query.json")
         .then()
         .statusCode(200)
-        .body("columns[0]", equalTo("employee_id"))
-        .body("rows.employee_id[0]",equalTo("1"));
-        //.body("columns[0]", equalTo("total_cnt"))
-        //.body("rows.total_cnt[0]",equalTo("1000"));
+        .body("columns[0]", equalTo("total_cnt"))
+        .body("rows.total_cnt[0]", equalTo("2"));
     //Get the response
     Response response = get("/profiles.json").then().extract().response();
     //Extract the entries for finishedQueries from the response
     List<String> jsonResponse = response.jsonPath().getList("finishedQueries");
     //Iterate through the repsonse to find the matching finished query
     int i;
-    for(i = 0; i <= jsonResponse.size(); i++) {
+    for (i = 0; i <= jsonResponse.size(); i++) {
       if (query.equalsIgnoreCase(response.jsonPath().param("i", i).getString("finishedQueries.query[i]"))) {
         break;
       }
       break;
     }
-    String queryId_1 = response.jsonPath().param("i", i).getString("finishedQueries.queryId[i]");
-//  }
-//  Dependencies in the tests!!!
-//  @Test
-//  public void verifyTestQueryProfileInfo() {
+    queryId_1 = response.jsonPath().param("i", i).getString("finishedQueries.queryId[i]");
+  }
+
+  @Test(dependsOnMethods = {"queryUpdatedTestPlugin"})
```

Comment:
I think "dependsOnMethods" should be changed to "priority", so you don't need to add it in every test.